### PR TITLE
docs: starting a container

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

Added procedure: _Starting a container_.

### CODE REVIEW

File: website/docs/getting-started/starting-a-container.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+# Starting a container on your Container engine
+
+With Podman Desktop, you can start a container from an image on your container engine.
+You can interact with the running container by using the terminal in Podman Desktop, or by opening your browser to the exposed ports.
+
+#### Prerequisites
+
+- The **<icon icon="fa-solid fa-cloud" size="lg" /> Images** list has your image, such as `docker.io/library/redis`, `docker.io/library/nginx`.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]:
```suggestion
- The **<icon icon="fa-solid fa-cloud" size="lg" /> Images** list has your image, such as `quay.io/podman/hello`, `docker.io/library/nginx`.
```

File: website/docs/getting-started/starting-a-container.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Procedure
+
+1. Go to **<icon icon="fa-solid fa-cloud" size="lg" /> Images**.
+1. On the line with your image name, click **<icon icon="fa-solid fa-play" size="lg" />**.
+1. On the **Create a container** screen, review the configuration.
+   1. **Container name**: enter a name, such as `my_container`.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: leave blank so it's autogenerated

File: website/docs/getting-started/starting-a-container.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+1. Click **<icon icon="fa-solid fa-play" size="lg" /> Start Container**.
+
+#### Verification
+
+1. Go to **<icon icon="fa-solid fa-cubes" size="lg" /> Containers**.
+1. Click your container name, such as `my_container`.
+1. To search logs:
+   1. Go to **Logs**.
+   1. Click the content area to activate it.
+   1. Enter `Ctrl +F` to start searching in the content.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: is that `<kbd>Ctrl</kbd>` is working ? would help to have `<kbd>Ctrl</kbd>+<kbd>F</kbd>`

also I'm not sure we can search in the logs anyway ?

File: website/docs/getting-started/starting-a-container.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+1. To inspect the container:
+   1. Go to **Inspect**.
+   1. Click the content area to activate it.
+   1. Enter `Ctrl +F` to start searching in the content.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: on macOS it's Cmd not Ctrl

File: website/docs/getting-started/starting-a-container.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+1. To generate Kubernetes YAML:
+   1. Go to **Kube**.
+   1. Click the content area to activate it.
+   1. Enter `Ctrl +F` to start searching in the content.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: Cmd on macOS

File: website/docs/getting-started/starting-a-container.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+   1. Copy the content to a YAML file.
+1. To interact with the container terminal:
+   1. Go to **Terminal**.
+   1. Click the content area to activate the terminal.
+   1. Enter your commands.
+1. To interact with the exposed ports:
+   1. Click **<icon icon="fa-solid fa-external-link" size="lg" />**.
+   1. Your browser opens a page to the exposed port on `localhost`.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: first exposed port

File: website/docs/getting-started/starting-a-container.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+1. Click your container name, such as `my_container`.
+1. To search logs:
+   1. Go to **Logs**.
+   1. Click the content area to activate it.
+   1. Enter `Ctrl +F` to start searching in the content.
+1. To inspect the container:
+   1. Go to **Inspect**.
+   1. Click the content area to activate it.
+   1. Enter `Ctrl +F` to start searching in the content.
+1. To generate Kubernetes YAML:
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: only if it's a podman engine, not a docker engine

File: website/docs/getting-started/starting-a-container.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+   1. Go to **Kube**.
+   1. Click the content area to activate it.
+   1. Enter `Ctrl +F` to start searching in the content.
+   1. Copy the content to a YAML file.
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md:41]: not sure to understand what to do ?

File: website/docs/getting-started/starting-a-container.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+1. To interact with the container terminal:
+   1. Go to **Terminal**.
+   1. Click the content area to activate the terminal.
+   1. Enter your commands.
+1. To interact with the exposed ports:
+   1. Click **<icon icon="fa-solid fa-external-link" size="lg" />**.
+   1. Your browser opens a page to the exposed port on `localhost`.
+1. To deploy to your current Kubernetes context:
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]: only if container engine is of podman engine type

File: website/docs/getting-started/starting-a-container.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 5
+title: Starting a container
+description: Starting a container on your container engine.
+keywords: [podman desktop, podman, containers, images]
+tags: [images]
+---
+
+# Starting a container on your Container engine
```
Comment (jeffmaury) [inline website/docs/getting-started/starting-a-container.md]:
```suggestion
# Starting a container on your container engine
```

File: website/docs/getting-started/starting-a-container.md (hunk 5 above)
Comment (benoitf) [reply website/docs/getting-started/starting-a-container.md]: the character for cmd is ⌘ : <kbd>⌘</kbd>+<kbd>F</kbd> on macOS

File: website/docs/getting-started/starting-a-container.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+1. On the line with your image name, such as `quay.io/podman/hello`, click **<icon icon="fa-solid fa-play" size="lg" />**.
+1. On the **Create a container** screen, review the configuration.
+1. Click **<icon icon="fa-solid fa-play" size="lg" /> Start Container**.
+
+#### Verification
+
+1. Go to **<icon icon="fa-solid fa-cubes" size="lg" /> Containers**.
+1. **<icon icon="fa-solid fa-search" size="lg" />**: Enter your image name, such as `quay.io/podman/hello`, to find your running container.
+1. Click your running container name.
+1. To search logs:
```
Comment (benoitf) [inline website/docs/getting-started/starting-a-container.md]:
```suggestion
1. To view logs:
```

view or browse but as we can't search might be misleading

**Comment by benoitf** [comment]
disabling auto-merge to allow other people to review (not a typo fix)

**Review by benoitf — APPROVED** [review]

**Pull request merged**